### ISSUE TITLE
Contribute code in Python

### ISSUE THREAD

**Issue by SAY-droid427** [SAY-droid427/DSAQuestins#22]

**Comment by SAY-droid427** [comment]
@96RadhikaJadhav @um4ng-tiw The previous issue got deleted. So I am creating this issue again. You may continue contributing code

**Comment by SAY-droid427** [comment]
@96RadhikaJadhav @um4ng-tiw Will you guys be contributing more code?

**Comment by um4ng-tiw** [comment]
I added all i could.

**Comment by SAY-droid427** [comment]
@um4ng-tiw Ok. I have also created a repository https://github.com/SAY-droid427/WebDevHub. Do check out its documentation.

**Comment by 96RadhikaJadhav** [comment]
> @96RadhikaJadhav @um4ng-tiw Will you guys be contributing more code?

Yes I want to add more. Thanks for the link :) . I will check it out.

**Comment by SAY-droid427** [comment]
@96RadhikaJadhav I am planning to help more students and particularly women in technology by providing them resources for studying computer science and information about various scholarships. So I have created a website for the same. Could you please share it among the people you know and help them? Here is the link: https://say-droid427.github.io/women_in_tech.html

**Comment by 96RadhikaJadhav** [comment]
> @96RadhikaJadhav I am planning to help more students and particularly women in technology by providing them resources for studying computer science and information about various scholarships. So I have created a website for the same. Could you please share it among the people you know and help them? Here is the link: https://say-droid427.github.io/women_in_tech.html

@SAY-droid427 Yeah Sure. I will.
